cloudweatherreport/model.py

```diff
@@ -544,7 +544,7 @@ def from_bundletester_output(cls, provider, output):
                 result_code = 'FAIL'
             result.tests.append(TestResult(
                 name=test['test'],
-                duration=test['duration'],
+                duration=test.get('duration', 0.0),
```

Comment:
Does 0 make sense as a default duration?  Perhaps it should just be `test.get('duration')`.  But, why would the duration ever be missing?